addons/html_builder/static/src/builder/snippets_menu_tabs/invisible_elements_tab.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+
+<t t-name="html_builder.InvisibleElementsTab">
+    <div class="o_we_invisible_el_panel">
+        <div class="o_panel_header">
+            Invisible Elements
+        </div>
+        <t t-foreach="state.invisibleEls" t-as="invisibleEntry" t-key="invisibleEntry_index">
+            <t t-call="html_builder.invisibleSnippetEntry" t-call-context="{'entry': invisibleEntry, 'invisibleElTab': this}"/>
```

Comment:
instead of `'invisibleElTab': this`, what about `'toggleElementVisibility': toggleElementVisibility`